lib/index.ts

```diff
@@ -35,43 +40,35 @@ try {
 }
 
 async function lock() {
-	return lockfile.check(LOCK_PATH, {}, async (err, isLocked) => {
-		if (err) {
-			return console.error(err.message);
-		}
+	try {
+		const isLocked = await checkLockAsync(LOCK_PATH);
 
 		if (!isLocked) {
 			console.log('applying lock...');
-			return lockfile.lock(LOCK_PATH, {}, async (e) => {
-				if (e) {
-					return console.error(e.message);
-				}
-				console.log('lock applied!');
-			});
+			await lockAsync(LOCK_PATH);
```

Comment:
Why do we need an await here but not on line 64?

Reply:
We should have it in both places, I missed the other - fixed now